runestone/activecode/activecode.py

```diff
@@ -126,7 +127,8 @@ class ActiveCode(Directive):
         'timelimit': directives.unchanged,
         'stdin' : directives.unchanged,
         'datafile' : directives.unchanged,
-        'sourcefile' : directives.unchanged
+        'sourcefile' : directives.unchanged,
+        'available_files' : directives.unchanged
```

Comment:
Yes. For example, see https://raw.githubusercontent.com/presnick/ProgramsInformationPeople/master/_sources/Assignments/ps3.rst  ps_3_7